owncloudApp/src/main/res/values/strings.xml

```diff
@@ -386,6 +386,8 @@
     <string name="placeholder_filesize">389 KB</string>
     <string name="placeholder_timestamp">2012/05/18 12:23 PM</string>
     <string name="placeholder_media_time">12:23:45</string>
+    <string name="file_image">File Image</string>
+    <string name="cancel_btn">Cancel download button</string>
```

Comment:
Good catch! Adding a content description is always important and definitely, it's something we should start doing.

At the moment, the `strings.xml` is a little messy, and we should try to add new values in proper places with some identifiers so we can know exactly which key is associated with which screen.

We already have some strings for this screen, they have the `filedetails_` prefix. Let's add these content descriptions next to them. So that, the keys could be `filedetails_file_image_content_description` and `filedetails_cancel_btn_content_description`. 👍

Reply:
Of course, I didn´t see them. I´m going to do it.